docs_src/microservices/application/AppServiceConfigurable.md

```diff
@@ -53,66 +30,105 @@ the configuration in the `configuration.toml` file under the
-`AppService-http-export` when using --profile option or just
-`AppService` if not using the --profile option.
+The first line of note is `ExecutionOrder = "FilterByDeviceName, TransformToXML, HTTPPost,MarkAsPushed"`. This specifies the order in which to execute your functions. Each function specified here must also be placed in the `[Writeable.Pipeline.Functions]` section. 
+
+Next, each function and its required information is listed. Each function typically has associated Parameters that must be configured to properly execute the function as designated by `[Writable.Pipeline.Functions.{FunctionName}.Parameters]`. Knowing which parameters are required for each function, can be referenced by taking a look at the `Transforms` documentation located in the [Application Functions SDK](./ApplicationFunctionsSDK.md#built-in-transformsfunctions) section. In a few cases, such as `TransformToXML`, `TransformToJSON`, or `SetOutputData`, there are no parameters required.
+
+
+> Note: By default, the configuration provided is set to use MessageBus as a trigger from CoreData. This means you must have EdgeX Running with devices sending data in order to trigger the pipeline. You can also change the trigger to be HTTP. For more on triggers, view the `Triggers`documentation located in the [Application Functions SDK](./ApplicationFunctionsSDK.md#triggers) section.
+
+That's it! Now we can run/deploy this service and the functions pipeline will process the data with functions we've defined.
+
+## Environment Variable Overrides For Docker
+
+Edgex service no longer has docker specific profiles. They now rely on environment variable overrides in the docker compose files for the docker specific differences. The following environment settings are required in the compose files when using App Service Configurable.
+
+```
+EDGEX_PROFILE : [target profile]
+SERVICE_HOST : [service name]
+SERVICE_PORT : [service port]
+MESSAGEBUS_SUBSCRIBEHOST_HOST: edgex-core-data
+CLIENTS_COREDATA_HOST: edgex-core-data
+```
+
+The following is an example docker compose entry for **App Service Configurable**:
+
+```yaml
+  app-service-configurable-rules:
+    image: edgexfoundry/docker-app-service-configurable:1.1.0
+    environment:
+      EDGEX_PROFILE: rules-engine
+      SERVICE_HOST: edgex-app-service-configurable-rules
+      SERVICE_PORT: 48096
+      MESSAGEBUS_SUBSCRIBEHOST_HOST: edgex-core-data
+      CLIENTS_COREDATA_HOST: edgex-core-data
+    ports:
+      - "48096:48096"
+    container_name: edgex-app-service-configurable-rules
+    hostname: edgex-app-service-configurable-rules
+    networks:
+      edgex-network:
+        aliases:
+          - edgex-app-service-configurable-rules
+    depends_on:
+      - data
+```
+
+> *Note: **App Service Configurable** is designed to be run multiple times each with different profiles. This is why in the above example the name `edgex-app-service-configurable-rules` is used for the instance running the `rules-engine` profile.*
```

Comment:
leverage admonition for "notes" (https://squidfunk.github.io/mkdocs-material/extensions/admonition/):
!!! note
    content

Reply:
Fixed in all instances